pkg/ccn-lite/ccn-lite-riot.h

```diff
@@ -12,6 +12,7 @@
 /**
  * @defgroup    pkg_ccnlite CCN-Lite stack
  * @ingroup     pkg
+ * @ingroup     net
```

Comment:
I agree with @aabadie here (which was, why I did not comment on that in the first place).